specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10773 @@
+      }
+    },
+    "CharFilterName": {
+      "type": "string",
+      "enum": [
+        "html_strip"
+      ],
+      "x-ms-enum": {
+        "name": "CharFilterName",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "html_strip",
+            "name": "HtmlStrip",
+            "description": "A character filter that attempts to strip out HTML constructs. See https://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/charfilter/HTMLStripCharFilter.html"
+          }
+        ]
+      },
+      "description": "Defines the names of all character filters supported by Azure Cognitive Search.",
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/rest/api/searchservice/Custom-analyzers-in-Azure-Search"
+      }
+    },
+    "RegexFlags": {
+      "type": "string",
+      "enum": [
+        "CANON_EQ",
+        "CASE_INSENSITIVE",
+        "COMMENTS",
+        "DOTALL",
+        "LITERAL",
+        "MULTILINE",
+        "UNICODE_CASE",
+        "UNIX_LINES"
+      ],
+      "x-ms-enum": {
+        "name": "RegexFlags",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "CANON_EQ",
+            "name": "CanonEq",
+            "description": "Enables canonical equivalence."
+          },
+          {
+            "value": "CASE_INSENSITIVE",
+            "name": "CaseInsensitive",
+            "description": "Enables case-insensitive matching."
+          },
+          {
+            "value": "COMMENTS",
+            "name": "Comments",
+            "description": "Permits whitespace and comments in the pattern."
+          },
+          {
+            "value": "DOTALL",
+            "name": "DotAll",
+            "description": "Enables dotall mode."
+          },
+          {
+            "value": "LITERAL",
+            "name": "Literal",
+            "description": "Enables literal parsing of the pattern."
+          },
+          {
+            "value": "MULTILINE",
+            "name": "Multiline",
+            "description": "Enables multiline mode."
+          },
+          {
+            "value": "UNICODE_CASE",
+            "name": "UnicodeCase",
+            "description": "Enables Unicode-aware case folding."
+          },
+          {
+            "value": "UNIX_LINES",
+            "name": "UnixLines",
+            "description": "Enables Unix lines mode."
+          }
+        ]
+      },
+      "description": "Defines flags that can be combined to control how regular expressions are used in the pattern analyzer and pattern tokenizer.",
+      "externalDocs": {
+        "url": "http://docs.oracle.com/javase/6/docs/api/java/util/regex/Pattern.html#field_summary"
+      }
+    },
+    "SearchFieldDataType": {
+      "type": "string",
+      "enum": [
+        "Edm.String",
+        "Edm.Int32",
+        "Edm.Int64",
+        "Edm.Double",
```

Comment:
Isn't this missing `Edm.Single` for vector fields?

Reply:
Vector fields are `Collection(Edm.Single)`. We don't expose single by itself right now